.github/workflows/codeql.yml

```diff
@@ -0,0 +1,101 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
I think you can drop all docs from the template

Reply:
will do